types/react-select/test/examples/AsyncCreatable.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
@andy-ms are our tests equipped to handle tests in a folder ?

Reply:
Yes, should work just fine.